Source/WebCore/style/PseudoElementIdentifier.h

```diff
@@ -37,4 +37,9 @@ struct PseudoElementIdentifier {
     AtomString pseudoElementNameArgument { nullAtom() };
 };
 
+inline static bool isStyleCacheable(const PseudoElementIdentifier& identifier)
+{
+    return identifier.pseudoElementNameArgument.isNull();
```

Comment:
`pseudoElementNameArgument` is an unnecessarily long name. How about just `argument`?

Reply:
I'll change to `nameArgument`, some pseudo-elements have arguments that are not names (e.g. `::part()`)